php-8.1-pecl-mongodb.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: php-8.1-pecl-mongodb
+  version: 1.16.2
+  epoch: 0
+  description: "PHP 8.1 MongoDB driver - PECL"
+  copyright:
+    - license: PHP-3.01
+  dependencies:
+    provides:
+      - php-pecl-mongodb=${{package.full-version}}
+    runtime:
+      - php-8.1
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - binutils
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - cyrus-sasl-dev
+      - gcc
+      - icu-dev
+      - libtool
+      - openssl-dev>3
+      - php-8.1-dev
+      - snappy-dev
+      - zstd-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://pecl.php.net/get/mongodb-${{package.version}}.tgz
+      expected-sha512: 3c81882c609b09cde534315aa4b1fe3c47e18e4ed26a940bf60a6bdbf4e53b2204d8e4e113a9b0a3469b60793ead9f8dff703920b86280e78448e07d6535a277
+
+  - name: phpize and configure
+    runs: |
+      phpize
+      ./configure --prefix=/usr --with-php-config=php-config
+
+  - uses: autoconf/make
+
+  - name: Install
+    runs: |
+      make INSTALL_ROOT="${{targets.destdir}}" install
+      install -d ${{targets.destdir}}/etc/php81/conf.d
+      echo "extension=mongodb" > ${{targets.destdir}}/etc/php81/conf.d/mongodb.ini
+
+  - uses: strip
+
+# TODO(vaikas): I can't find in release-monitoring, or github.
```

Comment:
I think it's here in release monitor https://release-monitoring.org/project/11158/.

The URL and version seems to match up at least?

Reply:
There it indeed!